### ISSUE TITLE
Folder element size grows with length of alias.

### ISSUE THREAD

**Issue by matthewcoming** [plantuml/plantuml#476]
code to reproduce:

```plantuml
@startuml TestFolderSize

folder "a small name" as ASmallName
folder "a small name" as ADecentlyLongAliasName
folder "smaller" as ADecentlyLongAliasNameButEvenLonger
folder ADecentlyLongAliasNameButEvenLongerAndReversedAliasOrdered as"smol"

@enduml
```

As you can see, the file element gets longer with the alias name. This behavior is different from all other elements, but at a glance this seems to be due to the shared code between package and folder elements.

**Comment by swmatisa** [comment]
Hello,
I already described thins in
https://forum.plantuml.net/12082/folder-size-change-with-alias-size-package-identifier-visible?show=12082#q12082

Le jeu. 4 mars 2021 à 00:47, Matthew Coming <notifications@github.com> a
écrit :

**Comment by matthew16550** [comment]
Fixed in #735 so will be available in the next release.
